lib/sms_text.rb

```diff
@@ -0,0 +1,17 @@
+require 'rubygems'
+class Sms_Text
+
+	def send_text
+		account_sid = 'AC1cffe35eb134133af61d8ab716f40588'
+		auth_token = '48e47a677c14dd9162f479523ed7fb3c'
```

Comment:
these shouldn't be on github - they give anyone access to your account. USe environment variables or dotenv instead.